integration/integration_test.go

```diff
@@ -3852,6 +3869,21 @@ func testRotateRollback(t *testing.T, s *integrationTestSuite) {
 
 	svc, err := waitForProcessStart(serviceC)
 	require.NoError(t, err)
+	defer svc.Shutdown(context.TODO())
```

Comment:
nit: I would prefer `context.Background()` unless you actually intend to go back and update this to plumb a context through.

Reply:
Actually removed those `Shutdown` calls from `testRotateSuccess` instead.